vue-frontend/src/App.vue

```diff
@@ -1,5 +1,13 @@
 <template>
-  <router-view></router-view>
+  <Teleport to="head">
+    <title>{{ seoData.title }}</title>
```

Comment:
Should we define teleport inside home page instead? That way we will not need setSeoData function.

Reply:
We need to define the same meta tags for 404 and contact screen also, that's why have taken it into app.vue to avoid duplication